From 06355fe2e15e5482df6e7a4b853596e3426a9918 Mon Sep 17 00:00:00 2001
From: Will Thompson <will@willthompson.co.uk>
Date: Mon, 15 Jul 2019 16:22:18 +0100
Subject: [PATCH 5/6] appdata: specify <translation>

This allows the generated appstream to include data on the app's
translation coverage.
---
 data/appdata/gnote.appdata.xml.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/appdata/gnote.appdata.xml.in b/data/appdata/gnote.appdata.xml.in
index 47edb72e..662740b2 100644
--- a/data/appdata/gnote.appdata.xml.in
+++ b/data/appdata/gnote.appdata.xml.in
@@ -24,5 +24,6 @@
  <releases>
   <release version="3.32.0" date="2019-03-17"/>
  </releases>
+ <translation type="gettext">gnote</translation>
 </application>
 
-- 
2.20.1

